canonical-web-and-design/ubuntu.com
# Fix outstanding vanilla issues

## Done

- fixed vertical alignment of social icons on blog posts
- fixed padding issue on secondary navigation
- fixed row/col layout in footer

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Go through the entire site and make sure everything looks OK
